datasets/fake_news_english/README.md

```diff
@@ -0,0 +1,151 @@
+---
+YAML tags:
+annotations_creators: [expert-generated]
+language_creators: [expert-generated]
+languages: [en]
+licenses: [unknown]
+multilinguality: [monolingual]
+size_categories: [n<1K]
+source_datasets: [original]
+task_categories: [text-classification]
+task_ids: [multi-label-classification]
+---
+
+# Dataset Card for Fake News English
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+- **Homepage: https://dl.acm.org/doi/10.1145/3201064.3201100** 
+- **Repository: https://github.com/jgolbeck/fakenews/**
+- **Paper: https://doi.org/10.1145/3201064.3201100 **
+- **Leaderboard: **
+- **Point of Contact: Jennifer Golbeck (http://www.jengolbeck.com)**
+
+### Dataset Summary
+
+### Supported Tasks and Leaderboards
+[More Information Needed]
+
+### Languages
+English
+
+## Dataset Structure
+
+### Data Instances
+{
+"article_number": "102",
+"url_of_article": "https://newslo.com/roger-stone-blames-obama-possibility-trump-alzheimers-attacks-president-caused-severe-stress/",
+"fake_or_satire": "Fake",
+"url_of_rebutting_article": "https://www.snopes.com/fact-check/donald-trumps-intelligence-quotient/"
+}
```

Comment:
Can you use backticks to put the example in a code block ?
For example
````
```
{
"article_number": "102",
"url_of_article": "https://newslo.com/roger-stone-blames-obama-possibility-trump-alzheimers-attacks-president-caused-severe-stress/",
"fake_or_satire": "Fake",
"url_of_rebutting_article": "https://www.snopes.com/fact-check/donald-trumps-intelligence-quotient/"
}
```
````

Reply:
also I'm pretty sure the article number is an integer (`102` instead of `"102"`) and fake_or_satire is an integer as well as it is a class label (`1` instead of `"Fake"` since class labels are encoded as integers)